.env.example

```
@@ -40,5 +40,8 @@ SUBGRAPH_WS_ENDPOINT=ws://127.0.0.1:8001/subgraphs/name/joinColony/subgraph
 # Used in non-local network deployments to customize the network address
 # NETWORK_CONTRACT_ADDRESS=
 
-Used (currently) to set a manual provider for Xdai deployments
+# Used (currently) to set a manual provider for Xdai deployments
 # RPC_URL=
+
+# Token for "userflow.com" user onboarding
+USERFLOW_TOKEN=ct_snrheuld3nbe3exppc4foxcb54
```

Please don't commit a "real" api key in the example environment variables

Just put in the variable's name:
```suggestion
USERFLOW_TOKEN=
```

---

Yeah, I think I just misunderstood your comment in the other related PR. Cleaning up now.